components/Chips/src/MDCChipView.m

```diff
@@ -250,6 +252,20 @@ - (void)dealloc {
                                                 object:nil];
 }
 
+- (void)setShapeGenerator:(id<MDCShapeGenerating>)shapeGenerator {
+  self.layer.shapeGenerator = shapeGenerator;
+
+  if (shapeGenerator) {
+    self.layer.cornerRadius = 0;
```

Comment:
Nuke the shadowPath as well?

Reply:
shadowPath gets nuked inside MDCShapedShadowLayer setShapeGenerator.

Reply:
Ah, k. That does feel like we're depending on an implementation detail that may change someday. Given that we're setting a shadowPath in this file I think it might be safer if we also clear it out here, but I could be missing some other aspect of this.

Reply:
You're right. Fixed.